services/user.py

```diff
@@ -0,0 +1,59 @@
+from db.models import User
+
+
+def create_user(
+    username: str,
+    password: str,
+    email: str = None,
+    first_name: str = None,
+    last_name: str = None
+) -> User:
+    user = User()
+    user.username = username
+    user.set_password(password)
+
+    if email:
+        user.email = email
+
+    if first_name:
+        user.first_name = first_name
+
+    if last_name:
+        user.last_name = last_name
+
+    user.save()
+    return user
+
+
+def get_user(user_id: int) -> User:
+    return User.objects.get(id=user_id)
+
+
+def update_user(
+    user_id: int,
+    username: str = None,
+    password: str = None,
+    email: str = None,
+    first_name: str = None,
+    last_name: str = None
+) -> None:
+    user = User.objects.get(id=user_id)
```

Comment:
The `get` method can raise a `DoesNotExist` exception if no user is found with the given ID. Consider handling this exception to prevent potential crashes.